backend/app/tests/api/routes/configs/test_version.py

```
@@ -483,3 +466,413 @@ def test_versions_isolated_by_project(
+
+    config_blob_result = data["data"]["config_blob"]
+    # Provider and type should be inherited
+    assert config_blob_result["completion"]["provider"] == "openai"
+    assert config_blob_result["completion"]["type"] == "text"
+    # Params should be updated
+    assert config_blob_result["completion"]["params"]["model"] == "gpt-4-turbo"
+    assert config_blob_result["completion"]["params"]["temperature"] == 0.9
+
+
+def test_create_config_with_kaapi_provider_success(
+    db: Session,
```

Ruff (0.14.14)

[warning] Unused function argument: `db`

(ARG001)